src/components/shifts/ShiftForm.vue

```diff
@@ -260,6 +280,24 @@ export default {
         this.contracts.find((contract) => contract.uuid === this.shift.contract)
       );
     },
+    contractShifts() {
+      return this.$store.getters["shift/shifts"].filter((shift) => {
+        return shift.contract.uuid === this.shift.contract.uuid;
+      });
+    },
+    sickOrVacationShift() {
+      if (this.shift === null) return undefined;
+      const shiftsOnSameDate = this.contractShifts.filter((shift) => {
+        return (
+          localizedFormat(parseISO(shift.date.start), "yyyy-MM-dd") ===
```

Comment:
es gibt eine `isSameDay`-Funktion (die auch schon in dieser Komponente importiert ist), ggf. kann oder sollte die hier benutzt werden.

Reply:
Good point, habe ich nicht dran gedacht!